src/project/internal/projectactionscontroller.h

```diff
@@ -1,26 +1,26 @@
 #ifndef AU_PROJECT_PROJECTACTIONSCONTROLLER_H
 #define AU_PROJECT_PROJECTACTIONSCONTROLLER_H
 
-#include "iprojectfilescontroller.h"
+#include "project/iprojectfilescontroller.h"
 
 #include "actions/actionable.h"
 #include "async/asyncable.h"
 
 #include "modularity/ioc.h"
-#include "iinteractive.h"
+#include "global/iinteractive.h"
```

Comment:
duplication

#include "global/iinteractive.h"

is already included three lines below